composer.json

```diff
@@ -20,12 +20,13 @@
         "composer/semver": "^1.4",
         "geerlingguy/ping": "^1.1",
         "illuminate/support": "5.2.*|5.3.*|5.4.*|5.5.*|5.6.*|5.7.*|5.8.*",
-        "vlucas/phpdotenv": "~2.5"
+        "vlucas/phpdotenv": "^3.3"
     },
     "require-dev": {
         "larapack/dd": "^1.0",
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "~3.5",
+        "orchestra/testbench": "^3.8",
+        "orchestra/testbench-core": "3.8.x-dev",
```

Comment:
This line can be removed once 3.8 is tagged for `orchestra/testbench-core`. See orchestral/testbench-core#21